src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordWithOcrTest.java

```diff
@@ -47,22 +55,66 @@
     private static final String RESPONSE_FIELD_WARNINGS = "warnings";
     private static final String RESPONSE_FIELD_DATA = "data";
     private static final String ATTACH_TO_CASE_REFERENCE_FIELD_NAME = "attachToCaseReference";
+    private static final String SERVICE_AUTHORIZATION_HEADER = "ServiceAuthorization";
+    private static final String BEARER = "Bearer";
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    private static final long EXCEPTION_RECORD_ID = 26409983479785245L;
+    private static final String EXCEPTION_RECORD_FILENAME = "record.pdf";
+    private static final String EXCEPTION_RECORD_DOCUMENT_NUMBER = "654321";
+
+    private static final Map<String, Object> EXCEPTION_RECORD_DOC = document(
+        EXCEPTION_RECORD_FILENAME,
+        EXCEPTION_RECORD_DOCUMENT_NUMBER
+    );
+
+    private static Map<String, Object> document(String filename, String documentNumber) {
+        return ImmutableMap.of(
+            "value", ImmutableMap.of(
+                "fileName", filename,
+                "controlNumber", documentNumber,
+                "someNumber", 3
+            )
+        );
+    }
```

Comment:
This method is among constants. Should be move to the bottom, below test methods.